api/README.md

```diff
@@ -0,0 +1,137 @@
+# API
```

Comment:
we need another section which talks about summary
- things to do like cloning
- configuration changes
- running ..etc